eth2util/eth2exp/attagg_test.go

```diff
@@ -42,16 +44,52 @@ func TestIsAggregator(t *testing.T) {
 	t.Run("aggregator", func(t *testing.T) {
 		// https://github.com/prysmaticlabs/prysm/blob/8627fe72e80009ae162430140bcfff6f209d7a32/beacon-chain/core/helpers/attestation_test.go#L26
 		commLen := uint64(3)
-		isAgg, err := eth2exp.IsAggregator(ctx, bmock, commLen, tblsconv.SigToETH2(blsSig))
+		isAgg, err := eth2exp.IsAttAggregator(ctx, bmock, commLen, tblsconv.SigToETH2(blsSig))
 		require.NoError(t, err)
 		require.True(t, isAgg)
 	})
 
 	t.Run("not an aggregator", func(t *testing.T) {
 		// https://github.com/prysmaticlabs/prysm/blob/fc509cc220a82efd555704d41aa362903a06ab9e/beacon-chain/core/helpers/attestation_test.go#L39
 		commLen := uint64(64)
-		isAgg, err := eth2exp.IsAggregator(ctx, bmock, commLen, tblsconv.SigToETH2(blsSig))
+		isAgg, err := eth2exp.IsAttAggregator(ctx, bmock, commLen, tblsconv.SigToETH2(blsSig))
 		require.NoError(t, err)
 		require.False(t, isAgg)
 	})
 }
+
+func TestIsSyncCommAggregator(t *testing.T) {
+	const (
+		syncCommSize = 64
+		nonAggSig    = "4837cabd917ad39f937e0b45afb6d5654f5742d118e4b98092107841e5cf44fc8601e3ce82d350eb0551ba844e4410f880a603dcc056fb46847a2a0568c4ed30f172ff57f591726c187e8ac228ab0f5651562048be9f60cfcefb8fac355369e5"
```

Comment:
please add links where you got the fixtures

Reply:
move variables to tests they are used in

Reply:
sure, i have changed the tests taking fixtures from https://github.com/prysmaticlabs/prysm/blob/39a7988e9edbed5b517229b4d66c2a8aab7c7b4d/beacon-chain/sync/validate_sync_contribution_proof_test.go#L336 and https://github.com/prysmaticlabs/prysm/blob/39a7988e9edbed5b517229b4d66c2a8aab7c7b4d/beacon-chain/sync/validate_sync_contribution_proof_test.go#L460